dev-util/maturin/maturin-1.5.0.ebuild

```diff
@@ -493,11 +495,12 @@ src_configure() {
 		--features full,password-storage$(usev ssl ,native-tls)
 	)
 
-	export MATURIN_SETUP_ARGS=${cargoargs[*]}
+	cargo_src_configure "${cargoargs[@]}"
```

Comment:
I recall I didn't use cargo_src_test for better control a while back, but with cargo_src_configure that saves arguments may as well use it, aka:
```diff
-edo cargo test $(usev !debug --release) ${MATURIN_SETUP_ARGS} -- "${skip[@]}"
+cargo_src_test -- "${skip[@]}"
```
or so (untested), and drop inherit edo

Reply:
(think I hadn't used it because I was handling USE=debug differently back then and didn't want it to add --release, but right now it's pointless either way given it is adding --release)

Reply:
Done and tested.